README.md

```diff
@@ -74,3 +82,18 @@ $ docker run \
 ```
 
 
+## Start GPU worker
+
+[nvidia installation instructions](https://github.com/NVIDIA/nvidia-docker#quickstart)
+
+```bash
+$ nvidia-docker run \
+    -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
add the /tmp/codalab-v2 volume here, too.

Reply:
added